libsolidity/codegen/ABIFunctions.cpp

```diff
@@ -1431,6 +1479,23 @@ string ABIFunctions::shiftRightFunction(size_t _numBits, bool _signed)
 	});
 }
 
+std::string ABIFunctions::scaleFixedFunction(int fromFractionalDigits, int toFractionalDigits, bool _signed)
+{
+	string functionName = "scale_fixed_" + toString(fromFractionalDigits) + "_to_" + toString(toFractionalDigits) + (_signed ? "_signed" : "_unsigned");
```

Comment:
Where is `toString` defined? Should've we use C++'s `to_string`?

Reply:
For those, since they are just `int`s, yes, we can use `to_string`. Slightly below, we have to use `toString` to get the `bigint` into a string.